docs/whats-new.asciidoc

```diff
@@ -2,7 +2,7 @@
 [chapter]
 = What's new in {minor-version}
 
-Here are the highlights of what’s new and improved in {elastic-sec}. For detailed information about this release, check out the <<release-notes, Release notes>>.
+Here are the highlights of what’s new and improved in {elastic-sec}. For detailed information about this release, check out the <<release-notes, Release notes>>. To discover how you can benefit from these enhancements, visit our https://www.elastic.co/blog/whats-new-elastic-security-8-10-0[blog post]. 
```

Comment:
Super small nit - I'd add this sentence after the first one since they're more related content-wise. Everything else looks good! I'll defer to @jmikell821 for the final approval, wording, etc. :)

```suggestion
Here are the highlights of what’s new and improved in {elastic-sec}. To discover how you can benefit from these enhancements, visit our https://www.elastic.co/blog/whats-new-elastic-security-8-10-0[blog post].  For detailed information about this release, check out the <<release-notes, Release notes>>. 
```